crates/ide-assists/src/handlers/inline_call.rs

```diff
@@ -363,16 +363,17 @@ fn inline(
         .collect();
 
     if function.self_param(sema.db).is_some() {
-        let this = || make::name_ref("this").syntax().clone_for_update().first_token().unwrap();
+        let this = || make::name_ref("this").syntax().clone_for_update().first_token();
```

Comment:
Let's keep this unwrap, it is impossible to fail

Reply:
Maybe turn it into an expect, so it's more clear it's intentional.

Reply:
Sure that works too